apps/plea/email.py

```diff
@@ -43,12 +43,17 @@ def send_plea_email(context_data):
 
     context_data: dict populated by form fields
     """
+
     try:
-        court_obj = Court.objects.get_by_urn(context_data["case"]["urn"])
-    except Court.DoesNotExist:
-        logger.error("URN does not have a matching Court entry: {}".format(
-            context_data["case"]["urn"]))
-        raise
+        case = Case.objects.get(
+            urn__iexact=context_data["case"]["urn"].upper(), sent=False)
+
+    except (Case.DoesNotExist, Case.MultipleObjectsReturned):
```

Comment:
This logic needs a deeper refactor at some point - there are issues with it. But agreed, for now i'm just going to make it do what it was doing previously, e.g. taking the first object.